rest-api-with-fastify/app.js

```diff
@@ -0,0 +1,83 @@
+import Fastify from 'fastify';
+import { createReadStream } from 'fs';
+import { resolve } from 'path';
+import { Romcal } from 'romcal';
+import * as GeneralRoman from '@romcal/calendar.general-roman';
+import { France_Fr as FranceFR } from '@romcal/calendar.france';
+
+const fastify = new Fastify({
+  logger: {
+    level: process.env.LOG_LEVEL || 'fatal',
```

Comment:
I can definitely do that 👍

Reply:
I usually validate the env var values though and use the default value if the value is invalid. :man_shrugging:

Reply:
yeah, i wasn't planning on doing that here since we're going with a fairly minimal implementation, and I didn't add the log level, i just moved the app (i.e. creating the application, before it is initialized and starts listening) to a standalone file for ease of testing